# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ctwm/fix-comment.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./windowbox.c.org	Tue Dec 11 16:38:52 2001
+++ ./windowbox.c	Mon Mar 17 11:50:44 2003
@@ -40,7 +40,7 @@
 {
     WindowBox *winbox;
 
-    //printf ("addWindowBox : name = %s, geometry = %s\n", boxname, geometry);
+/*    printf ("addWindowBox : name = %s, geometry = %s\n", boxname, geometry); */
     winbox = (WindowBox*) malloc (sizeof (WindowBox));
     winbox->next     = NULL;
     winbox->name     = strdup (boxname);
